paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityFriction.java

```diff
@@ -0,0 +1,85 @@
+package com.denizenscript.denizen.paper.properties;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import io.papermc.paper.entity.Frictional;
+import net.kyori.adventure.util.TriState;
+
+public class EntityFriction implements Property {
+
+    public static boolean describes(ObjectTag entity) {
+        return entity instanceof EntityTag
+                && ((EntityTag) entity).getBukkitEntity() instanceof Frictional;
+    }
+
+    public static EntityFriction getFrom(ObjectTag entity) {
+        if (!describes(entity)) {
+            return null;
+        }
+        return new EntityFriction((EntityTag) entity);
+    }
+
+    private EntityFriction(EntityTag ent) {
+        entity = ent;
+    }
+
+    EntityTag entity;
+
+    @Override
+    public String getPropertyString() {
+        Boolean frictionState = ((Frictional) entity.getBukkitEntity()).getFrictionState().toBoolean();

```

Comment:
Should have a `getFrictional` method to do the cast, both for cleaner code and as that's the common practice with other modern entity properties